AUTHORS

```diff
@@ -21,18 +21,18 @@ bini11
 Brandon Ebersohl
 Brian Warner
 byrw
+César Carruitero
```

Comment:
This file is updated when we tag a new release.  There's no need to include it here.

Reply:
@chenba  Plzzz tell the script to undo this file.